Build not working on Mac OS 10.14.2

I get the following errors after build.sh:

```
-- Build files have been written to: /Users/artem/git/cb-multios/build
[465/11140] Linking C shared library include/tiny-AES128-C/libtiny-AES128-C.dylib
FAILED: include/tiny-AES128-C/libtiny-AES128-C.dylib
: && /usr/bin/clang  -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk -dynamiclib -Wl,-headerpad_max_install_names -m32 -o include/tiny-AES128-C/libtiny-AES128-C.dylib -install_name @rpath/libtiny-AES128-C.dylib include/tiny-AES128-C/CMakeFiles/tiny-AES128-C.dir/aes.c.o   && :
ld: warning: The i386 architecture is deprecated for macOS (remove from the Xcode build setting: ARCHS)
ld: warning: ignoring file /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/lib/libSystem.tbd, missing required architecture i386 in file /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/lib/libSystem.tbd
Undefined symbols for architecture i386:
  "___memset_chk", referenced from:
      _AES128_CBC_encrypt_buffer in aes.c.o
      _AES128_CBC_decrypt_buffer in aes.c.o
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
[474/11140] Building C object challenges/HeartThrob/CMakeFiles/HeartThrob_patched.dir/src/check.c.o
ninja: build stopped: subcommand failed.
```

--- a/README.md
+++ b/README.md
@@ -33,7 +33,17 @@ This is a helper script to test all challenges using `cb-test`. Results are summ
 
 The following steps will build both the patched and unpatched binaries in `build/challenges/[challenge]/`.
 
-### OS X/Linux
+### MacOS
+
+The challenges build as i386 binaries, but Mac OS 10.14+ only supports building x86-64 binaries by default. To enable i386 support, run the following command:
+
+```
+sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /
+```
+
+After this, proceed to the common directions for MacOS and Linux.
+
+### Linux
 
 The following packages are required for building the challenges on Linux:
 
@@ -41,11 +51,15 @@ The following packages are required for building the challenges on Linux:
 libc6-dev libc6-dev-i386 gcc-multilib g++-multilib clang cmake
 ```
 
+### MacOS/Linux Common Directions
+
+First, install pre-requisites via pip.
+
 ```
 sudo pip install xlsxwriter pycrypto defusedxml pyyaml matplotlib
 ```
 
-To build all challenges, run:
+Then to build all challenges, run:
 
 ```bash
 $ ./build.sh

